src/components/calcite-label/calcite-label.scss

```diff
@@ -13,71 +13,69 @@
 }
 
 :host([scale="s"]) {
-  --calcite-label-margin-bottom: 12px;
+  --calcite-label-margin-bottom: theme("spacing.3");
```

Comment:
this CSS var is used once, which makes it kind of weird, IMO.

I would just add this margin to your label declaration:


```diff
:host([scale="s"]) {
-  --calcite-label-margin-bottom: theme("spacing.3");
  label {
-  @apply text--2;
+  @apply text--2 mb-3;
  }
```